src/sentry/api/endpoints/builtin_symbol_sources.py

```diff
@@ -18,6 +19,7 @@ def normalize_symbol_source(key, source):
 
 @region_silo_endpoint
 class BuiltinSymbolSourcesEndpoint(Endpoint):
+    owner = ApiOwner.OWNERS_INGEST
```

Comment:
```suggestion
    owner = ApiOwner.OWNERS_NATIVE
```

Symbol sources should be part of native I believe